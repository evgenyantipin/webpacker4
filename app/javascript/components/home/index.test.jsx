import React from 'react'
import { shallow } from 'enzyme'
import Home from '.'

it('Home', () => {
  const output = shallow(<Home />)
  expect(output).toMatchSnapshot()
})
