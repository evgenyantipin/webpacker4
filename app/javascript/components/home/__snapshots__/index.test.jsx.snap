// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home 1`] = `
<div>
  Homepage
</div>
`;
